website/landing/components/common/Card.tsx

```diff
@@ -0,0 +1,67 @@
+import { styled } from "../../stitches.config";
```

Comment:
There are a few parts of the landing that have a very similar styling, I have extracted it to a single place and plan on applying it where it's appropriate.